<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsosurfaceConfig</class>
 <widget class="QWidget" name="IsosurfaceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>412</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min Scalar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scalars</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max Scalar</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of Contours</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="openFileButton">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QSpinBox" name="numOfContoursSpinBox"/>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="maxScalarSpinBox">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="minScalarSpinBox">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="displayEffectCombo">
       <item>
        <property name="text">
         <string>Transparent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Opaque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Display Effect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="scalarCombo"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Gradient</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="Widgets::GradientComboBox" name="gradientCombo"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::GradientComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/gradientcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
